eoscostarica/eos-rate
# fix(webapp): add filters and styling compare tool

### Add filters and styling compare tool

# What does this PR do?
- Add filter for Block Producer list page
- Add Styling for compare tool component
- Add responsive for Polar Radar Graph 

### Steps to test

1. run `make run`
1. go to `localhost:3000`
1. check Block Producers page

### Checklist
